src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/Consent.cshtml

```diff
@@ -0,0 +1,78 @@
+﻿@page
+@model FamilyHubs.Referral.Web.Pages.ProfessionalReferral.ConsentModel
+@{
+}
+
+@section Back {
+    <a href="/ProfessionalReferral/ConnectFamilyToServiceStart?id=@Model.Id&name=@Model.Name" class="govuk-back-link">Back</a>
+}
+
+<form method="post">
+    <input asp-for="@Model.Name" type="hidden">
+    <input asp-for="@Model.Id" type="hidden">
+    <div class="govuk-form-group">
+        <div class="govuk-form-group ">
+
+            @if (!Model.ValidationValid)
+            {
+                <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" data-module="govuk-error-summary">
+                    <h2 class="govuk-error-summary__title" id="error-summary-title">
+                        There is a problem
+                    </h2>
+                    <div class="govuk-error-summary__body">
+                        <ul class="govuk-list govuk-error-summary__list">
+                            <li>
+                                <a href="#service-name" data-testid="enter-service-name-error">Select whether you have permission from the family to share details.</a>
```

Comment:
Should this be an anchor?

Reply:
That is the format in the GDS as far as I know

Reply:
What component needs this to me a link?
Could you send me a link to the appropriate GDS page please.
[Radio controls](https://design-system.service.gov.uk/components/radios/) don't need it as far as I can see.